fdb-record-layer-core/src/main/proto/record_metadata.proto

```diff
@@ -48,7 +48,6 @@ message Index {
     RANK = 3;
     RANK_UNIQUE = 4;    // (Meaning the score is unique.)
   };
-  // TODO: Generalize to predicate on records in storage area to be included in index.
```

Comment:
That comment is from 2015.

Reply:
Was it rightful to remove it now that we have a `Predicate` member of `Index`?

Reply:
Seems fine. The generalized predicate implements the spirit of the comment, I believe.